www/members/nominations.cgi

```diff
@@ -37,7 +37,7 @@ def setup_data(cur_mtg_dir)
 
   # parse nominations for names and ids
   nominations = IO.read(File.join(cur_mtg_dir, 'nominated-members.txt').untaint).
-    scan(/^---+--\s+(.*?)\n/).flatten
+    scan(/^---+--\s+([a-z]+)\s+(.*?)\n/).flatten
```

Comment:
This assumes that the availid is alphanumeric characters from one to unbounded followed by a space or many spaces.
But availids (historical) might contain special characters "." or "_" and numbers. Should the regex be changed from
[a-z]+
to 
[a-z._0-9]+
to accommodate the dozens of availids that contain special characters?